File: AddressBook.Web/app/model/person.ts
export class Person {
    Id: number;
    FirstName: string;
    LastName: string;
    Address: string;
    Designation: string;
    Phone: string;
    Email: string;
    Job: string;
    Institution: string;

    constructor(id: number, fname: string, lname: string, address: string, dsgn: string, phone: string, email: string,job:string,institution:string) {
        this.Id = id;
        this.FirstName = fname;
        this.LastName = lname;
        this.Address = address;
        this.Designation = dsgn;
        this.Phone = phone;
        this.Email = email;
        this.Job = job;
        this.Institution = institution;
    }
}

File: AddressBook.Core/Model/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AddressBook.Core.Model
{
    public class Person
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }

        public string FullName
        {
            get { return string.Format("{0} {1} {2}", this.FirstName, this.MiddleName, this.LastName); } 
        }

        public string Email { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public string Job { get; set; }
        public string Institution { get; set; }
    }
}
